app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    comment = Comment.new(text: params[:text], user: current_user, post:)
+    respond_to do |format|
+      format.html do
+        if comment.save
+          comment.counter_updater
+          flash[:success] = 'New comment added'
+        else
+          flash.now[:error] = 'Error: Comment could not be saved'
+        end
+        redirect_to user_post_comments_path
+      end
+    end
+  end
```

Comment:
- [ ] Please add a `new` method for creating a new comment and add its route too as the project requirements like you did in post controller you added both new and create method.

![image](https://user-images.githubusercontent.com/68796232/178997068-9b1257d3-40c0-42ba-9169-29d30fade4d2.png)